Tools/clinic/clinic.py

```diff
@@ -6427,8 +6451,12 @@ def format_docstring_parameters(params: list[Parameter]) -> str:
     def format_docstring(self) -> str:
         assert self.function is not None
         f = self.function
-        if f.kind.new_or_init and not f.docstring:
-            # don't render a docstring at all, no signature, nothing.
+        k = f.kind
+        if (k.new_or_init or k in {GETTER, SETTER}) and not f.docstring:
+            # If following cases have a empty docstring,
+            # Do not render a docstring at all, no signature, nothing.
+            # 1. __init__ / __new__
+            # 2. @getter / @setter properties
```

Comment:
I think there's a change in behaviour with this logic. If `f` is not a getter nor a setter, the old behaviour should stay the same; however, with this logic, there is a behaviour change.

I think my suggestion to consolidate these was a bad choice. Let's revert that. Sorry for the added work!